Header header
int32 count
SegmentedObject[] objects
